packages/hop-node/src/vault/Vault.ts

```diff
@@ -1,11 +1,9 @@
-import { AaveVault } from './AaveVault'
 import { BigNumber, Signer } from 'ethers'
 import { Chain } from 'src/constants'
 import { YearnVault } from './YearnVault'
 
 export enum Strategy {
   Yearn = 'yearn',
-  Aaave = 'aave'
```

Comment:
considering we've never used this feature, does it make sense to also remove yearn and the concept of vaults in hop node?

Reply:
Removed this along with some other unused code. Can reference this if we ever reintroduce it.